pkg/apis/broker/v1beta1/broker_validation.go

```diff
@@ -28,14 +28,22 @@ import (
 func (b *Broker) Validate(ctx context.Context) *apis.FieldError {
 	// We validate the GCP Broker's delivery spec. The eventing webhook will run
 	// the other usual validations.
-	return ValidateDeliverySpec(ctx, b.Spec.Delivery).ViaField("spec", "delivery")
+	if b.Spec.Delivery == nil {
+		return apis.ErrMissingField("spec.delivery")
+	}
```

Comment:
Now we require delivery spec?

Reply:
The backoff retry policy defaulting creates the delivery spec. If we don't require delivery spec, then we don't require backoff retry policy, and the backoff retry policy defaulting/validation will be duplicated between the webhook and the trigger reconciler, and default backoff settings will need to be carried in code in addition to in the ConfigMap. I assumed that it would be best to handle all the defaulting through the ConfigMap.

Reply:
Make sense.

Reply:
After speaking with @grantr , it seems to make more sense to allow for nil delivery spec in validation, but validate a non-nil delivery spec, and also add a Broker SetDefaults in the reconciler if the delivery spec is nil. Disallowing a nil delivery spec could break existing Brokers for users who update Knative. My latest commit reflects this, so I've unresolved the conversation to see if this is alright with you.

Thanks so much for reviewing this, by the way!